### ISSUE TITLE
RFC logo

### ISSUE THREAD

**Issue by tiborsimko** [reanahub/www.reana.io#8]
REANA has been without a proper logo for too long.

We have been musing about various options on paper, basically expressing "re"-usable "ana"-lyses concept, keeping closeness to CAP, and introducing some leading illustrative glyph or picture that could resemble computational workflow concept (=triangled graph) or the reuse concept (=recycle arrow) and the like.  The leading glyph/illustration could be later used for favicon.

Here is the representation prepared by @jbenito3:

![logo-reana](https://user-images.githubusercontent.com/517546/40067925-a27c5e42-5867-11e8-99fa-fa8c1c345c5e.png)

Are we happy with the idea and with the prepared logo "as is"?  Any minor suggestions regarding  appearance? Any major suggestions with conceptually different proposals?

**Comment by hjhsalo** [comment]
Looks very good!

I have to say though, that to my eye it looks like spacing between letters in `re` and `ana` is a bit off.
`re` part is ok, but the first `a` in `ana` is too close to `re` -part.
I might move `re` and `ana` a little bit further apart or reduce the space between letters of `ana` -part. Maybe the latter.

On the other hand, logo looks very good as is on the website mockup (https://github.com/reanahub/reana.io/issues/9).

**Comment by diegodelemos** [comment]
I really like the result! I am all in for going with this version 😄

**Comment by dinosk** [comment]
I like it too, I agree with @hjhsalo regarding the kerning.
Two nitpicky fixes would be:

to center the top red square with the intersecting lines, there is a little less on the left side:

<img width="617" alt="screen shot 2018-05-16 at 09 48 43" src="https://user-images.githubusercontent.com/1566093/40103664-888493ce-58ee-11e8-9573-ddcb29c80949.png">

and to align the lines of e and a (not possible without chaning the a -> making it incosistent with CAP, but looks a bit off right now):

<img width="915" alt="screen shot 2018-05-16 at 09 52 36" src="https://user-images.githubusercontent.com/1566093/40103773-e390a2a8-58ee-11e8-9de5-f88c9a711171.png">

I'm happy to help if I can.

**Comment by jbenito3** [comment]
Thank you all for your feedback. Just one comment regarding spacing between letters: it was done on purpose that `re` is connected (like workflows are connected through steps) to `ana`. `ana` was kept as is in *CAP* logo as a reference to the service. So the metaphor in there is: the "connected" `re` sort of represents the connected workflows (steps) towards our CAP `ana`lyses. 
Separating them with the same spacing everywhere will give yet another classical textual logo (personallly I tried it and I didnt like it...).

**Comment by jbenito3** [comment]
@hjhsalo @dinosk I have reduced the spaces in `ana` and doing a better alignment for both the workflow and the `e` with `a`. See:

![reana1](https://user-images.githubusercontent.com/717470/40185075-0dbbe34a-59f2-11e8-9e77-e0df213052a3.png)

Is that any better?

I went crazy and started experimenting a bit further...but I do not like it:

![reana2](https://user-images.githubusercontent.com/717470/40185106-2301361a-59f2-11e8-89a7-46a5992dd6e4.png)

**Comment by dinosk** [comment]
@jbenito3 yes imo it looks better! I vote for the first revision

**Comment by jbenito3** [comment]
More options following some of your ideas:

![reana-recycle](https://user-images.githubusercontent.com/717470/40241513-8218c5ea-5abb-11e8-9417-f94c961e4334.png)

---

![reana3](https://user-images.githubusercontent.com/717470/40241520-859bc078-5abb-11e8-8485-6adac19833da.png)

---

![reana4](https://user-images.githubusercontent.com/717470/40241524-880032fe-5abb-11e8-81cf-4468a6bacbb2.png)

---

![reana5](https://user-images.githubusercontent.com/717470/40241539-90a74ec4-5abb-11e8-96ad-3bf6a8f34097.png)

**Comment by tiborsimko** [comment]
FWIW from the first batch I like the option number one, from the second batch the option number two. 

(Perhaps making the leading image ten per cent smaller and slightly more separated from the text, so that it would stand out without blending with the main logo text...  Otherwise we risk people seeing words like "areana" for the first batch and "oreana" for the second batch?)

@reanahub/developers Any other proposals, ideas, observations? Let's aim at closing this issue later this week...

**Comment by lukasheinrich** [comment]
I also vote for number two of the second batch

**Comment by dinosk** [comment]
I also like better option number two from the second batch

**Comment by atrisovic** [comment]
Oh wow, I like how 'ana' has it's own font 😁
I vote for no. 4 in the second batch.

**Comment by atrisovic** [comment]
I am not 100% sold on the colours (but I don't really have better ideas 😅)

**Comment by atrisovic** [comment]
Also maybe 'r' and 'e' can have a bit of space in between?

**Comment by jbenito3** [comment]
Ok, the third batch with somebody else's ideas:

![reana6](https://user-images.githubusercontent.com/717470/40494017-315643ea-5f74-11e8-9215-a1e9af3aa61f.png)

---


![reana7](https://user-images.githubusercontent.com/717470/40494020-31871bc8-5f74-11e8-9dfc-51dbad65ee2a.png)

---


![reana8](https://user-images.githubusercontent.com/717470/40494021-31a50098-5f74-11e8-8834-3f3c64bd373f.png)

---


![reana9](https://user-images.githubusercontent.com/717470/40494022-31c18736-5f74-11e8-986b-86e99955cd74.png)

---


![reana10](https://user-images.githubusercontent.com/717470/40494023-31e49cf8-5f74-11e8-8cd6-a59351f6bdda.png)

---


![reana11](https://user-images.githubusercontent.com/717470/40494025-321c20c4-5f74-11e8-9762-dccd098e3fbe.png)

---


![reana12](https://user-images.githubusercontent.com/717470/40494027-3252df60-5f74-11e8-85e4-07372eff7cee.png)

**Comment by TimSmithCH** [comment]
Despite them showing great creativity... I feel that they all look too familiar and evoke the wrong images - the third set are ying/yang, the second set are recycling. So the only one that looks new/unique and therefore distinctive for a new service is the first one, the branching workflows...

**Comment by suenjedt** [comment]
It is true that they do look familiar. I don't find it too bad though. I like first batch, 1st one and Ying/Yang

**Comment by tiborsimko** [comment]
It can be considered a good thing if the logo looks *just a little bit* familiar, since it may make people feel comfortable with the product via gentle subliminal messages :wink:  Obviously, if the similarity is *too obvious* or *too crude*, then it wouldn't work so well.

E.g. from the third batch the second logo may evoke also two swimming koi carps around each other, or two gravitating galaxies, as it were... so the resemblance to yin/yang is more on the gentle side and the logo might work. While the first logo may look perhaps too close to a Pepsi, and the sixth logo too close to a tennis ball, so these logos might work less well.

**Comment by costaflavio** [comment]
I like some general/common aspects of all the logo: in particular nice colors and size/proportions, good job!
I also like a lot the "recycle" idea which is well espressed by the recycle logo (in one of the sets) and the font for "re".
The "analysis" idea unfortunately does not seem surfacing to me. For that I would consider:
- replacing the dot in the center of the recycle icon with a glass flask (e.g. 
![image](https://user-images.githubusercontent.com/7466334/40767630-5675ea1a-64b3-11e8-9e49-704fe87ef8b6.png) ) or a magnifying lense (which reminds of Invenio, too)
- using for "ana" a more "scientific looking"(?) font (not sure what to suggest, perhaps Century)

**Comment by TimSmithCH** [comment]
Great point Flavio - whilst a pure recycle symbol looks too "familiar", a modified one with a science symbol in the middle would indeed match Tibor's "just a little bit familiar" criterion :)   So it could be a flask, or an atom, or anything else scientific looking - what's the most _familiar_ science symbol ?!?

**Comment by diegodelemos** [comment]
I think that [first batch first version](https://github.com/reanahub/reana.io/issues/8#issuecomment-389893568) still delivers the clearest message, expressing **how** REANA allows physicists to produce/reproduce scientific outputs, using the abstract idea of a graph (even if it is not all what REANA can do, but it is its essence).

However, the [recycle batch](https://github.com/reanahub/reana.io/issues/8#issuecomment-390232101) refers to **what** REANA does in a very generic or "familiar" way as mentioned before. Besides the fact that this could be fixed by Flavio's idea, I think that it refers to the result, whereas the graph (or first batch first version) refers to the process, which in my opinion is more transcendental, a workflow/graph helps to create better structured analysis making reproducibility easy.

To sum up, it is more a "philosophy/way of working" versus a "result/what you get" difference. For example, you could use REANA to make your analysis reproducible and make all the analysis in one step, everything in one go. It would be reproducible, REANA will use the right data|code|environment, but it will be more difficult to understand for a new collaborator.

**Comment by tiborsimko** [comment]
We seem to like the textual part of the logo, so I guess this one is settled. I propose to simply deploy the textual part of the logo today while the graphics part of the logo is being discussed and various ideas graphically polished. This will enable deployment, the textual logo looks already nice, and using it will give us time to explore the other options in calm next week.

(Besides we could even consider sticking to the textual version only :wink: like Google and others)

**Comment by jbenito3** [comment]
@tiborsimko I did a pull request with your suggestion. In my eyes I miss the logo, maybe I grew too attached to the simple workflow logo, and also now we are even closer to nextflow...see:

![screencapture-localhost-5000-2018-06-01-09_56_06](https://user-images.githubusercontent.com/717470/40828860-1d8c1c68-6582-11e8-93ef-b3334b7025ee.png)

**[UPDATE]**

Having a second look...maybe it is not that bad... :) I guess I need some minutes to re-adjust...

**Comment by suenjedt** [comment]
I like it without logo, in fact. More zen.

**Comment by tiborsimko** [comment]
Closing the RFC as the textual logo is being used.  In case we would like to revive leading graphics, we can open a new one.
